src/t8_geometry/t8_geometry_helpers.c

```diff
@@ -125,78 +125,97 @@ t8_geom_compute_linear_geometry (t8_eclass_t tree_class, const double *tree_vert
       t8_geom_triangular_interpolation (ref_coords + offset_tree_dim, tri_vertices, T8_ECLASS_MAX_DIM, 2,
                                         out_coords + offset_domain_dim);
     }
-  } break;
-  case T8_ECLASS_LINE:
-  case T8_ECLASS_QUAD:
-  case T8_ECLASS_HEX:
-    for (coord = 0; coord < num_coords; coord++) {
-      const size_t offset_tree_dim = coord * dimension;
-      const size_t offset_domain_dim = coord * T8_ECLASS_MAX_DIM;
-      t8_geom_linear_interpolation (ref_coords + offset_tree_dim, tree_vertices, T8_ECLASS_MAX_DIM, dimension,
-                                    out_coords + offset_domain_dim);
-    }
-    break;
-  case T8_ECLASS_PYRAMID: {
-    double base_coords[2];
-    double vec[3];
-    for (coord = 0; coord < num_coords; coord++) {
-      const size_t offset_tree_dim = coord * dimension;
-      const size_t offset_domain_dim = coord * T8_ECLASS_MAX_DIM;
-      /* Pyramid interpolation. After projecting the point onto the base,
+    t8_geom_triangular_interpolation (ref_coords + offset_tree_dim, tri_vertices, T8_ECLASS_MAX_DIM, 2,
+                                      out_coords + offset_domain_dim);
+  }
+  }
+  break;
+case T8_ECLASS_LINE:
+case T8_ECLASS_QUAD:
+case T8_ECLASS_HEX:
+  for (coord = 0; coord < num_coords; coord++) {
```

Comment:
As in the previous PR. Prepend `i` in front of looping variables.
```suggestion
  for (icoord = 0; icoord < num_coords; icoord++) {
```